src/main/java/com/igot/cb/playlist/service/impl/PlayListServiceImpl.java

```diff
@@ -126,6 +127,11 @@ public ApiResponse createPlayList(JsonNode playListDetails) {
       enrichedContentJson.put(Constants.ID, jsonNodeEntity.getOrgId());
       persistInRedis(enrichedContentJson, jsonNodeEntity, jsonNodeEntity.getOrgId()+jsonNodeEntity.getRequestType());
       JsonNode playListJson = jsonNodeEntity.getData();
+      List<String> searchTags = new ArrayList<>();
+      searchTags.add(playListJson.get(Constants.TITLE).textValue().toLowerCase());
+      searchTags.add(playListJson.get(Constants.TYPE).textValue().toLowerCase());
+      ArrayNode searchTagsArray = objectMapper.valueToTree(searchTags);
+      ((ObjectNode) playListJson).putArray("searchTags").add(searchTagsArray);
```

Comment:
Get this constant value from Constant files searchTags

Reply:
made changes